shared_crates/ui/src/scroll_area.rs

```diff
@@ -32,14 +38,29 @@ pub fn ScrollArea(
     /// The scroll area sizing
     sizing: ScrollAreaSizing,
     /// The child element
-    inner: Element,
+    inner: Element
 ) -> Element {
     let (scroll, set_scroll) = hooks.use_state(0.);
     let (inner_size, set_inner_size) = hooks.use_state(Vec2::ZERO);
-    hooks.use_runtime_message::<messages::WindowMouseWheel>(move |_world, event| {
+    let res = use_window_logical_resolution(hooks);
+    let max_scroll_down = match sizing {
```

Comment:
There's a MeasureSize component we should use to figure out what the size of the ScrollArea is, instead of supplying it from outside